lib/santa_maria/gateway/santa_maria_legacy.rb

```diff
@@ -48,7 +46,18 @@ def variants_for(global_id)
 
       private
 
-      attr_reader :endpoint
+      attr_reader :endpoint, :site_code, :language
+
+      def get(uri)
+        request = Net::HTTP::Get.new(uri)
```

Comment:
Net::HTTP is so grim

Reply:
Ha, yeah, we did spend some time with Kyle trying to make it work for v2 gateway ;fff

Reply:
But that's a gem – better to keep deps low.

Reply:
:+1: agreed